client/widgets/CArtifactsOfHeroBackpack.cpp

```diff
@@ -84,6 +95,7 @@ void CArtifactsOfHeroBackpack::scrollBackpack(int offset)
 		slot = slot + 1;
 	}
 	redraw();
+	setRedrawParent(true);
```

Comment:
should be in constructor I believe

Reply:
is it not supposed to be used everytime to notify parent UI element that redraw is currently needed? @IvanSavenko will know better

Reply:
No. This sets persistent flag that will be used in redraw() call. So 
- should be called before redraw(), not after
- can be called just once, for example in constructor